torch/_dynamo/symbolic_convert.py

```diff
@@ -949,6 +949,61 @@ def COMPARE_OP(self, inst):
                     **options,
                 )
             )
+        elif (
+            isinstance(left, BaseListVariable)
+            and all(
+                isinstance(l, (ConstantVariable, DynamicShapeVariable))
+                for l in left.items
+            )
+            and isinstance(right, BaseListVariable)
+            and all(
+                isinstance(r, (ConstantVariable, DynamicShapeVariable))
```

Comment:
suggest a helper method to handle this repeated "all constant or dynamic shape" check